From fa624e3f88602792012d343d41e80e2008429743 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Thu, 2 Sep 2010 14:59:44 +0000
Subject: [PATCH 24/63] gettext tests: remove old sanity test under NO_GETTEXT
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Remove an old sanity test that checked that the share/locale dir
didn't exist if we compiled with NO_GETTEXT=YesPlease. This would
break if the user did:

    git clean -dxf
    make NO_GETTEXT=
    make NO_GETTEXT=YesPlease
    make test

This test was something to test very basic sanity back when I was
first developing the gettext series, but it's not needed anymore. We
don't have similar tests for other parts of git to check that we
didn't build something with a given feature disabled.

Reported-by: Junio C Hamano <gitster@pobox.com>
Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 t/t0200-gettext-basic.sh |    5 -----
 1 files changed, 0 insertions(+), 5 deletions(-)

diff --git a/t/t0200-gettext-basic.sh b/t/t0200-gettext-basic.sh
index 3846ea8..8853d8a 100755
--- a/t/t0200-gettext-basic.sh
+++ b/t/t0200-gettext-basic.sh
@@ -39,11 +39,6 @@ test_expect_success GETTEXT 'sanity: Icelandic locale was compiled' '
     test -f "$TEXTDOMAINDIR/is/LC_MESSAGES/git.mo"
 '
 
-test_expect_success NO_GETTEXT "sanity: \$TEXTDOMAINDIR doesn't exists with NO_GETTEXT=YesPlease" '
-    ! test -d "$TEXTDOMAINDIR" &&
-    test "$TEXTDOMAINDIR" = "$GIT_TEXTDOMAINDIR"
-'
-
 # TODO: When we have more locales, generalize this to test them
 # all. Maybe we'll need a dir->locale map for that.
 test_expect_success GETTEXT_LOCALE 'sanity: gettext("") metadata is OK' '
-- 
1.7.2.3.313.gcd15

